README.md

```diff
@@ -1,17 +1,17 @@
-# 🇺🇸 English (US)
+# Fabulously Optimized
 
-Welcome to the Fabulously Optimized wiki!
+Welcome to the Fabulously Optimized wiki! [🌐 Read in your language](https://fabulously--optimized-gitbook-io.translate.goog/modpack/?_x_tr_sl=auto&_x_tr_tl=et&_x_tr_hl=et&_x_tr_pto=wapp)
```

Comment:
Ideally this link would detect the user's preferred language but I am not sure if I can use JS in this.

Reply:
Fixed with https://webapps.stackexchange.com/a/12126 with slight modifications (https and removal of target language)